README.md

```diff
@@ -48,10 +47,22 @@ Options:
 -c  continuous             If true, ignores -n and -v and will continuously send queries to the destination and will log results
 ```
 
+## Generating traces
+To generate synthetic traces: 
+```bash
+$ ./load-test traces -f test.json -t 1000 -s 10
```

Comment:
can you also add a note outlining how two different `./load-test ingest ...` commands are needed (one for service & one for spans)?